foundation/foundation-data-access/pom.xml

```diff
@@ -98,8 +98,26 @@
                     <groupId>xml-apis</groupId>
                     <artifactId>*</artifactId>
                 </exclusion>
+            <exclusion>
```

Comment:
I: These 3 items have IA issues, and aren't needed in the context of aissemble (several of these have been dropped entirely in hive 4.0)